logging/README.md

```diff
@@ -45,21 +40,23 @@ There are two methods to create a local copy of the repository:
 
 #### Download a Compressed Copy of the Repository
 
-1. On the [Releases](https://github.com/sassoftware/viya4-monitoring-kubernetes/releases) page, locate the release that you want to deploy.
-2. Expand **Assets** for the release, which is located below the release notes.
-3. Select either **Source code (.zip)** or **Source code (.tar.gz)** to download the repository 
+1. On the main page of the repository, click on Releases (on the right side of the repository contents area) to display the [Releases](https://github.com/sassoftware/viya4-monitoring-kubernetes/releases) page. 
+2. Locate the release that you want to deploy. Typically, you should download the latest release, which is the first one listed.
+3. Expand **Assets** for the release, which is located below the release notes.
+4. Select either **Source code (.zip)** or **Source code (.tar.gz)** to download the repository 
 as a compressed file.
-4. Expand the downloaded file to create a local copy of the repository. The repository is created
+5. Expand the downloaded file to create a local copy of the repository. The repository is created
 in a directory named `viya4-monitoring-kubernetes-<release_number>`.
 
 #### Clone the Repository
 
-1. From the main page for the repository, click **Code**.
-2. Copy the HTTPS URL for the repository.
-3. From a directory where you want to create the local copy, enter the 
-command `git clone <https_url>`. 
-4. Change to the `viya4-monitoring-kubernetes` directory.
-5. Enter the command `git checkout <release_number>`
+1. From the main page for the repository, select the **stable** branch, which is the most recent officially released version. The **master** branch is the branch under active development.
+2. From the main page for the repository, click **Code**.
+3. Copy the HTTPS URL for the repository.
+4. From a directory where you want to create the local copy, enter the 
+command `git clone <https_url>`. If you need to use a repeatable process to download a specific release of the repository, include the tag associated with the release in the `git clone` command. The tag for each release is displayed on the Releases page. 
```

Comment:
In Step 4, if you want to include a specific branch/tag/release in the git clone command, the syntax is:
git clone --branch <my_branch> <https_url>
That still clones the whole repo but does a git checkout my_branch behind the scenes.  
This would work for cloning the stable branch or a specific release (although the option is called branch, you can specify a tag with it as well.  This would eliminate the need for Step 6.

Change the sentence about repeatable process to something like:  _If you are developing a repeatable process and need to ensure the same release of the repo is used each time, specify the tag associated with a specific release rather than **stable**. Note that the tag and release names are typically the same but you should confirm the tag by looking at the Releases page._